ha/alert/filter.py

```diff
@@ -57,9 +58,9 @@ def __init__(self):
         super().__init__()
 
         # Get filter type and resource types list from the alert monitor rule file
-        self.alert_filter_components_list = Conf.get(const.ALERTS.PK_ALERT_EVENT_COMPONENTS)
-        self.alert_filter_modules_list = Conf.get(const.ALERTS.PK_ALERT_EVENT_COMPONENT_MODULES)
-        self.alert_filter_module_operations = Conf.get(const.ALERTS.PK_ALERT_EVENT_OPERATIONS)
+        self.alert_filter_components = Conf.get(ALERT_EVENT_INDEX, const.ALERTS.PK_ALERT_EVENT_COMPONENTS)
```

Comment:
You can import the ALERTS module from const and use that directly

Reply:
Done